pkgs/development/python-modules/aws-encryption-sdk/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, wrapt
+, cryptography
+, attrs
+, boto3
+}:
+
+buildPythonPackage rec {
+  pname = "aws-encryption-sdk";
+  version = "3.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qwmgwzhakb8xdby97vafg3rcll0zlvx0kdymhf6p3f6iw0vypwd";
+  };
+
+  propagatedBuildInputs = [
+    wrapt
+    cryptography
+    attrs
+    boto3
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  meta = {
+    description = "AWS Encryption SDK implementation for Python";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ sree ];
+    homepage = "https://github.com/aws/aws-encryption-sdk-python";
+  };
+
+  doCheck = false;
```

Comment:
Disabling checks and including pytestCheckHook is pretty misleading. Please only do one or the other on the future.